rwengine/src/render/OpenGLRenderer.hpp

```diff
@@ -14,23 +14,41 @@ typedef uint64_t RenderKey;
 
 typedef std::uint32_t RenderIndex;
 
-struct VertexP3 {
+// just a handle
+struct VertexP {
```

Comment:
What is this for?

Reply:
I was thinking it - "@todo normalize this to have the same interface as VertexP3" can mean there's possibility that in future one vector will store VertexP2 and VertexP3.